README_new.md

```diff
@@ -0,0 +1,132 @@
+* It should compile and run.
+
+## 2. Already written project
+
+We would be happy if you would submit a project you already have (e.g. potentially a demo project for another company). Still  the project has to obey to the following requirements:
+
+* Use Swift 3.0.
+* It should have at least two distinct network calls.
+* It should parse the network response and present the information to the user.
+* It should have some sort of persistence mechanism.
+* It should compile and run.
+* **Ideally** it should have a point of synchronization (e.g. making two concurrent requests and waiting for both of them to finish).
+
+If you have a project with these requirements then perfect! Please ensure you have a description of what the project does, in order to give us some context. ❤️
+
+## 3. Open Source work
+
+We would like to see a **non-trivial** pull request you have done to a public open source project. This should be something you are proud and where you show your technical skills. **It should also be related to iOS development**: it should be aligned with what you will do on a day-to-day basis. 😊✨🌳
+
+---
+
+## General Tips/Advice
+
+* We like code that is simple, [but simple is different than easy](https://www.infoq.com/presentations/Simple-Made-Easy).
+* Keep in mind the [SOLID principles](https://en.wikipedia.org/wiki/SOLID_(object-oriented_design)) when doing the project.
+* We did leave out of the requirements if you should try to download everything (posts + each post detail). This is up to you to decide and justify why you did so.
+* Testing is very important for us. Even if you don't write a single test (e.g. because of time constraints), your app should be easy to test (we love [Dependency injection](https://en.wikipedia.org/wiki/Dependency_inversion_principle)).
+* Error scenarios should be taken into consideration as well and how easy is to add them, even if you don't explicitly show them (e.g. using an `UIAlertController`).
+* Although UI and UX are important, in this demo, we are more concerned on how you architect your application and your thought process. It should take into consideration features that might be added in the future.
+* You can use any 3rd party library you wish (Alamofire, ReactiveCocoa, PromiseKit, Realm, etc). In case you do, be prepared to justify why you used it. You can use CocoaPods, Carthage or Git Submodules for this (please don't drag and drop stuff around).
+* **Be consistent with your code**. We advise using something like [raywenderlich's swift style guide](https://github.com/raywenderlich/swift-style-guide) while doing the demo. It's absolutly fine to use any other style, as long as you are consistent.
+* Clean the file project structure (if you are creating one from scratch) and remove any unused methods (e.g AppDelegate). This shows attention to detail.
+
+#### Thanks for your time, we look forward to hearing from you!
+- The [babylon health iOS team](http://github.com/Babylonpartners)
```

Comment:
Why do we think that opening our GitHub main page inspires candidates to apply?

Reply:
Ditto.